app/service/rest.py

```diff
@@ -453,16 +453,16 @@ def get_all_notifications_for_service(service_id):
         data = notifications_filter_schema.load(MultiDict(request.get_json()))
         current_app.logger.debug(f"use POST, request {request.get_json()} data {data}")
 
-    if data.get("to"):
-        notification_type = (
-            data.get("template_type")[0] if data.get("template_type") else None
-        )
-        return search_for_notification_by_to_field(
-            service_id=service_id,
-            search_term=data["to"],
-            statuses=data.get("status"),
-            notification_type=notification_type,
-        )
+    # if data.get("to"):
+    #     notification_type = (
+    #         data.get("template_type")[0] if data.get("template_type") else None
+    #     )
+    #     return search_for_notification_by_to_field(
+    #         service_id=service_id,
+    #         search_term=data["to"],
+    #         statuses=data.get("status"),
+    #         notification_type=notification_type,
+    #     )
```

Comment:
I'm assuming this was in support of email notifications?  If so, let's just remove it!